bitwarden_license/src/Scim/Groups/PatchGroupCommand.cs

```diff
@@ -49,15 +53,15 @@ public async Task PatchGroupAsync(Guid organizationId, Guid id, ScimPatchModel m
                 else if (operation.Path?.ToLowerInvariant() == "displayname")
                 {
                     group.Name = operation.Value.GetString();
-                    await _groupService.SaveAsync(group, EventSystemUser.SCIM);
+                    await _updateGroupCommand.UpdateGroupAsync(group, EventSystemUser.SCIM);
```

Comment:
I don't think we're supposed to call commands from other commands. But I don't know a better solution for now without revisiting this command entirely, so I think it's fine for now.